cli/lib/kontena/cli/helpers/health_helper.rb

```diff
@@ -49,5 +49,49 @@ def node_health(node, grid_health)
         return node['connected'] ? :ok : :offline
       end
     end
+
+    # @return [Symbol]
+    # @return [String]
+    def node_etcd_health(node_health)
```

Comment:
Probably not good idea to pollute helper with resource specific helpers? Although we have done this in the past...